src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/PayRatesService.kt

```diff
@@ -158,34 +153,28 @@ class PayRatesService(
     )
   }
 
-  private fun publishTelemetry(
-    courseActivityId: Long,
```

Comment:
This is now published along with its parent so we just build the telemetry in this service